python/GafferUI/Backups.py

```diff
@@ -373,6 +378,18 @@ def __backupNumberEnabled( plug ) :
 
 		],
 
+		"backups.enableAutoLoad" : [
+
+			"description",
+			"""
+			Toggles autoloading backups on Gaffer start. If enabled
+			and there exists a backup then Gaffer will hang and ask
+			the user if they want to open the backup instead. However,
+			if disabled then Gaffer will skip the backup.
```

Comment:
This applies any time a file is opened, not just on startup. And _hang_ is a very negative term - what we're actually doing is prompting the user to see if they would like to open the backup. Perhaps something like this :

> When a file is opened, checks for a more recent backup. If one is found, a dialogue is displayed, asking if the backup should be opened instead. If this option is turned off, you can still access backups via the `File/Open Backup` menu item.